packages/application-extension/src/index.tsx

```diff
@@ -362,6 +366,61 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isEnabled: () => !labShell.isEmpty('right')
       });
 
+      commands.addCommand(CommandIDs.activateSidebarOverlays, {
+        label: trans.__('Show Side bar overlays'),
+        execute: args => {
+          console.log('correct');
+          if (args.side != 'left' && args.side != 'right') {
+            throw Error(`Unsupported sidebar: ${args.side}`);
+          }
+
+          const widgets = Array.from(labShell.widgets(args.side));
+          for (let index = 0; index < widgets.length; index++) {
+            let elementId = widgets[index].id;
+            let focusElement = document.querySelector(
+              "[data-id='" + elementId + "']"
+            );
+
+            if (focusElement && !focusElement.classList.contains('overlay')) {
+              const options: Overlay.IOptions = {
+                hostElement: focusElement,
+                classConfig: {
+                  class: 'JP-Overlay',
+                  title: 'JP-Overlay-Tooltip'
+                },
```

Comment:
I think these can be defaults for `Overlay` no need to specify these here.

Reply:
Moved these to creation of the overlay div.